src/layouts/BaseLayout.js

```diff
@@ -53,49 +53,140 @@ import GeographicalFeaturesInfrastructureLayout from '../GeographicalFeatures/la
-  '/': 'Home',
-  '/alerts': 'Alerts',
-  '/alerts/alerts': 'All Alerts',
-  '/alerts/actions': 'Actions Taken',
-  '/alerts/feeds': 'Feeds',
-  '/alerts/feedback': 'Surveys & Feedback',
-  '/alerts/sources': 'Data Sources',
-  '/alerts/servicerequests': 'Service Requests',
-  '/assessments': 'Assessments',
-  '/assessments/indicators': 'Indicators',
-  '/assessments/questionnaires': 'Questionnaires',
-  '/assessments/questions': 'Questions',
-  '/assessments/responses': 'Responses & Observations',
-  '/incidents': 'Incidents',
-  '/incidents/incidenttypes': 'Incidents Types',
-  '/incidents/commandcenter': 'Command Center',
-  '/incidents/assessments': 'Assessments',
-  '/incidents/lossdatabase': 'Loss Database',
-  '/incidents/feeds': 'Feeds',
-  '/incidents/actions': 'Actions',
-  '/geographicalfeatures': 'Geographical Features',
-  '/geographicalfeatures/administrativeboundaries': 'Administrative Boundaries',
-  '/geographicalfeatures/facilities': 'Facilities',
-  '/geographicalfeatures/infrastructure': 'Critical Infrastructure',
-  '/geographicalfeatures/warehouses': 'Warehouses',
-  '/plans': 'Emergency Plans',
-  '/plans/planner': 'Planner',
-  '/plans/activations': 'Activations',
-  '/plans/drills': 'Drills & Exercises',
-  '/plans/disseminations': 'Disseminations',
-  '/resources': 'Resources',
-  '/resources/stock': 'Stock',
-  '/resources/items': 'Items',
-  '/resources/adjustments': 'Adjustments',
-  '/resources/utilization': 'Utilizations',
-  '/resources/warehouses': 'Warehouses',
-  '/stakeholders': 'Stakeholders',
-  '/stakeholders/contacts': 'Contacts',
-  '/stakeholders/roles': 'roles',
-  '/stakeholders/notifications': 'Notifications',
+  '/': { name: 'Home', title: 'Emis' },
+  /* Alerts components */
+  '/alerts': { name: 'Alerts', title: 'Alerts module' },
+  '/alerts/actions': {
+    name: 'Actions Taken',
+    title: 'List of all perfomed actions',
+  },
+  '/alerts/feedback': {
+    name: 'Surveys & Feedback',
+    title: 'Alerts surveys and feedback',
+  },
+  '/alerts/feeds': { name: 'Feeds', title: 'Alerts feeds' },
+  '/alerts/alerts': { name: 'All Alerts', title: 'List of all alerts' },
+  '/alerts/servicerequests': {
+    name: 'Service Requests',
+    title: 'Alerts service requests',
+  },
+  '/alerts/sources': {
+    name: 'Data Sources',
+    title: ' Data Sources of alerts',
+  },
+  /* end of Alerts components */
+  /* Assessments components */
+  '/assessments': { name: 'Assessments', title: 'Assessments module' },
+  '/assessments/indicators': {
+    name: 'Indicators',
+    title: 'Indicators for monitoring assissment',
+  },
+  '/assessments/questionnaires': {
+    name: 'Questionnaires',
+    title: 'List of questionnaires',
+  },
+  '/assessments/questions': {
+    name: 'Questions',
+    title: 'List of questions',
+  },
+  '/assessments/responses': {
+    name: 'Responses & Observations',
+    title: 'Assissment Responses and observations',
+  },
+  /* end of assissment components */
```

Comment:
there is a typo here

Reply:
Also I have a suggestion 

we can use separators like this
```js
/* Assessments routes */
  '/assessments': { name: 'Assessments', title: 'Assessments module' },
  '/assessments/indicators': {
    name: 'Indicators',
    title: 'Indicators for monitoring assissment',
  },
  '/assessments/questionnaires': {
    name: 'Questionnaires',
    title: 'List of questionnaires',
  },
  '/assessments/questions': {
    name: 'Questions',
    title: 'List of questions',
  },
  '/assessments/responses': {
    name: 'Responses & Observations',
    title: 'Assissment Responses and observations',
  },

/* Geographical Routes */
...
```

let remove `end of something`  line

Reply:
Also there are many typos in this file ... 
it is supposed to be `assessments` and not `assissment`

Reply:
Also on the separation comments please use `routes` and not `components` since these are routes